### DISCUSSION TITLE
[Q] Rich Text Format

### DISCUSSION THREAD

**Discussion by BenMaGit** [exceljs/exceljs#2431] [Q&A]
## 💬 Questions and Help

Hello, I am trying to parse an xlsx file containing styled text and hyperlinks.
After following the instruction using 
`ws.getCell('A1').value` 
to read a cell that looks like this:
![Screen Shot 2020-07-20 at 2 06 20 PM](https://user-images.githubusercontent.com/45992016/87906605-86e03580-ca95-11ea-8b70-d823f952327d.png)

This is what I get: 

```lang===javascript
const rtf = {
  text: {
    richText: [
  {
  font: { name: 'Helvetica', color: { argb: 'FF545454' } },
  text: 'Helvetica'
  },
  { font: { bold: true, color: { argb: 'FF545454' } }, text: 'Bold, ' },
  {
    font: { bold: true, italic: true, color: { argb: 'FF545454' } },
    text: 'Bold+Italic\nNext Line italic and ',
  },
  {
    font: { bold: true, italic: true, color: { argb: 'FFFF0000' } },
    text: 'Red',
  },
  {
    font: { bold: true, italic: true, color: { argb: 'FF545454' } },
    text: '\n',
  },
  { font: { color: { argb: 'FF545454' } }, text: '\n' },
  {
    font: { color: { argb: 'FF1155CC' }, underline: true },
    text: 'Link Here',
  },
],
  },
  hyperlink: 'http://google.com/',
};
```

1. Is there a way to map the link to the text?
2. Is the richText output grouped by the font and styled of the text? 
     I am asking this because I have no idea how to use this format to write back to a file and have the same result
3. Is there a way to parse a cell value into html instead? (This is actually what I want as the end result)

Thanks for the help!

**Comment by sagarpomal** [comment]
@BenMaGit I'm trying the same. Did you find any solution?

**Comment by BenMaGit** [comment]
Unfortunately no.

**Comment by Ks89** [comment]
I have the same problem. Is there any news?
